omnibus/config/software/datadog-cf-finalize.rb

```diff
@@ -30,11 +30,6 @@
             copy "#{cf_source_root}/agent/agent.exe", "#{cf_bin_root_bin}"
             copy "#{cf_source_root}/agent/libdatadog-agent-three.dll", "#{cf_bin_root_bin}"
 
-            unless windows_arch_i386?
-              copy "#{cf_source_root}/agent/install-cmd.exe", "#{cf_bin_root_bin}/agent"
-              copy "#{cf_source_root}/agent/uninstall-cmd.exe", "#{cf_bin_root_bin}/agent"
-            end
```

Comment:
These commands are actually used in the cloudfoundry build: since the output is not an MSI but a ZIP, we need to provide a way to run the changes that the installer would do.